operator/cmd/mesh/install.go

```diff
@@ -136,12 +138,17 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 	if err != nil {
 		return err
 	}
+	setFlags := applyFlagAliases(iArgs.set, iArgs.manifestsPath, iArgs.revision)
 	// Ignore the err because we don't want to show
 	// "no running Istio pods in istio-system" for the first time
 	_ = DetectIstioVersionDiff(cmd, tag, kubeClient, iArgs)
 	// Warn users if they use `istioctl install` without any config args.
 	if !rootArgs.dryRun && !iArgs.skipConfirmation {
-		prompt := fmt.Sprintf("This will install the Istio %s profile into the cluster. Proceed? (y/N)", tag)
+		profile, enabledComponents, err := getProfileAndEnabledComponents(setFlags, iArgs.inFilenames, iArgs.force, iArgs.kubeConfigPath, iArgs.context, l)
+		if err != nil {
+			return fmt.Errorf("failed to get profile and enabled components: %v", err)
+		}
+		prompt := fmt.Sprintf("This will install the Istio %s %s profile with %q components into the cluster. Proceed? (y/N)", tag, profile, enabledComponents)
```

Comment:
If I use `profile=empty` the output looks like this.

```
$ ./out/linux_amd64/istioctl install --set profile=empty --set hub=gcr.io/istio-testing -d manifests/
! Istio control planes installed: 1.7.3.
! Use --revision or --force to install Istio.
This will install the Istio 1.9.0 empty profile with [] components into the cluster. Proceed? (y/N)
```
I think in case of an empty profile we should just say?

```
$ ./out/linux_amd64/istioctl install --set profile=empty --set hub=gcr.io/istio-testing -d manifests/
! Istio control planes installed: 1.7.3.
! Use --revision or --force to install Istio.
This will install the Istio 1.9.0 empty profile into the cluster. Proceed? (y/N)
```

Reply:
Thanks for the reminder, fixed.